website/Rakefile

```diff
@@ -17,7 +17,8 @@ task :test do
 
         /jstorm.io/,
         /datatorrent.com/,
-        /ai.google/ # https://issues.apache.org/jira/browse/INFRA-16527
+        /ai.google/, # https://issues.apache.org/jira/browse/INFRA-16527
+        /globenewswire.com/
```

Comment:
Can you add a comment as to why this is failing? Preferably a JIRA that we can can track context and check up on to see when it is fixed.

Reply:
done